.github/workflows/knative-releasability.yaml

```diff
@@ -40,15 +41,27 @@ jobs:
           go-version: 1.15.x
 
       - name: Install Dependencies
-        run: |
-          GO111MODULE=on go get tableflip.dev/buoy@master
+        run: GO111MODULE=on go get tableflip.dev/buoy@master
 
       - name: Check out code
         uses: actions/checkout@v2
+        with:
+          path: ./src/knative.dev/${{ github.event.repository.name }}
 
       - name: Check
-        run: |
-          buoy check go.mod --release $RELEASE --verbose
+        run: buoy check go.mod --release ${RELEASE} --domain knative.dev --verbose
+
+    - name: Upgrade
+      run: ./hack/update-deps.sh --release ${RELEASE} --upgrade
+
+    - name: Verify
+      run: |
+        if [[ -z "$(git status --porcelain)" ]]; then
+            echo "${{ github.repository }} up to date."
+        else
+            echo "${{ github.repository }} is out of date."
+            exit 1
+        fi
```

Comment:
I think we can use Markus' trick: https://github.com/mattmoor/knobots-actions/issues/7 to predicate the slack notification and customize its message without this *failing*

Reply:
done